data-lineage/src/main/java/com/wipro/ats/bdre/lineage/api/PersistenceUnit.java

```diff
@@ -197,12 +176,12 @@ public void populateBeans(UniqueList<Table> finalInTableNodes, UniqueList<Table>
 			lineageConstantNodes.add(node);
 		}
 		// one query
-		lineageQueryInfo = new LineageQueryInfo();
+		/*lineageQueryInfo = new LineageQueryInfo();
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) This block of commented-out lines of code should be removed. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonar.openbdre.org/coding_rules#rule_key=squid%3ACommentedOutCodeLine)